src/components/Chat/ChatMessage.tsx

```
@@ -0,0 +1,69 @@
+      className={`flex ${isOwnMessage ? 'justify-end' : 'justify-start'} mb-4`}
+    >
+      <div
+        className={`max-w-[70%] p-4 rounded-lg ${
+          isOwnMessage
+            ? 'bg-blue-500 text-white'
+            : 'bg-white border border-gray-200 dark:bg-gray-800 dark:border-gray-700'
+        }`}
+      >
+        <div 
+          className="prose dark:prose-invert max-w-none"
+          dangerouslySetInnerHTML={{ __html: message.content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)